src/phoenix/server/api/types/Dataset.py

```diff
@@ -236,6 +252,115 @@ async def experiments(
             ]
         return connection_from_list(data=experiments, args=args)
 
+    @strawberry.field
+    async def compare_experiments(
+        self,
+        info: Info[Context, None],
+        baseline_experiment_id: GlobalID,
+        comparison_experiment_ids: List[GlobalID],
+    ) -> List[ExperimentComparison]:
+        OrmExample = models.DatasetExample
+        OrmExperiment = models.Experiment
+        OrmRun: TypeAlias = models.ExperimentRun
+        OrmRevision = models.DatasetExampleRevision
+        OrmTrace = models.Trace
```

Comment:
hoist outside of function scope?

Reply:
addressed in 8a5e2450bc330620190667acdb0801f48b5a3479